README.md

```diff
@@ -31,7 +31,7 @@ to all Giant Swarm installations.
   - v26.1
     - [v26.1.0](https://github.com/giantswarm/releases/tree/master/capa/v26.1.0)
   - v26.0
-    - [v26.0.0](https://github.com/giantswarm/releases/tree/master/capa/v26.0.0)
+    - [v26.0.0](https://github.com/giantswarm/releases/tree/master/capa/archived/v26.0.0)
```

Comment:
shoudn't we move this to archived folder like in Vintage?

Reply:
Isn't this what Alex is doing? The `archived` folder is in the provider folder.

Reply:
Yes, the archived folder is in the provider directory. The README.md file still needs a link to the release in the new directory.